charters/PM/charter.md

```diff
@@ -0,0 +1,83 @@
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+### Project Lead
+
+When a DCP charter is approved, the community explictly delegates authority and leadership for a specific project and its scope to a **Project Lead**. The **Project Lead** is responsible for clearly defining and communicating the long-term vision and direction of their project.
+
+Similar to [Mozilla module owners](https://www.mozilla.org/en-US/about/governance/policies/module-ownership/), **Project Leads** are never *tyrants*. They are chartered to make decisions for their DCP project with input from the community and in the best interests of the community. 
+
+### Product Owner
+As adapted from the [Scrum process](https://www.scrumguides.org/scrum-guide.html#team-po), a **Product Owner** manages the [Product Backlog](https://www.scrumguides.org/scrum-guide.html#artifacts-productbacklog) for their DCP project including:
+>* Clearly expressing Product Backlog items
+>* Ordering the items in the Product Backlog to best achieve goals and missions
+>* Optimizing the value of the work the Development Team performs
+>* Ensuring that the Product Backlog is visible, transparent, and clear to all, and shows what the Scrum Team will work on next
+>* Ensuring the Development Team understands items in the Product Backlog to the level needed
+
+**NOTE**: The responsibilities of the **Project Lead** and **Product Owner** are tightly intertwined. At an overly simplistic level, the **Project Lead** is more responsible for strategic objectives while the **Product Owner** drives tactical execution. In actual practice, it is a close partnership when planning and coordinating development with other projects.
+
+### Science Program Manager
+
+The **Science Program Manager** is the formal liaison for requests between HCA Science Leadership and the DCP community. 
+
+*Current Science PM*: [Jonah Cool](jcool@chanzuckerberg.com)
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
+
+*Communication*
+* Coordinate regular outreach and announcements to the DCP community and the HCA Science Leadership
+
+* Primary liaison between the HCA Science Leadership and the DCP community facilitated by the *Science Program Manager*
+
+*Planning*
+
+Strategic 
+
+* Identify and maintain a repository of user personas and stories informed by *DCP UX* research
```

Comment:
I have added an issue with a proposed solution on how to do this https://github.com/HumanCellAtlas/dcp-community/issues/47